Web/Edubase.Web.UI/Areas/Establishments/Views/Establishment/Partials/TabHelpdesk.cshtml

```diff
@@ -24,21 +24,21 @@
             {
                 <div class="govuk-summary-list__row">
                     <dt id="edubase-trigger-1" class="govuk-summary-list__key">EduBase trigger 1</dt>
-                    <dd class="govuk-summary-list__value">@Model.Establishment.HelpdeskTrigger1</dd>
+                    <dd id="edubase-trigger" class="govuk-summary-list__value">@Model.Establishment.HelpdeskTrigger1</dd>
```

Comment:
Does this need to be `edubase-trigger-1` as you added to one of the other ids yesterday or is it the only one so does not need numbering?

Reply:
this is the value of the edubase trigger 1 to differentiate both I used edubase trigger